src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,90 @@
+package Duke;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+
+
+ /**
+  * <pre>
+  * Stroage class, 
+  * this class is to deal with 
+  * loading tasks from the file and saving tasks in the file.
+  * </pre>
+  * @param <String> Filepath path to the data txt file
+  */
+public class Storage {
+
+    String filepath;
```

Comment:
Perhaps you could consider using camelCase for all variable names to comply with the coding standard!